tests/randomized/lib/templates/docker-compose.template.yml

```diff
@@ -8,7 +8,7 @@ services:
   agent:
     image: datadog/agent:7
     environment:
-      - DD_API_KEY=${DATADOG_API_KEY}
+      - DD_API_KEY=${DATADOG_API_KEY:-not_a_valid_api_key}
```

Comment:
Recent versions of the datadog agent won't even start without a non-empty `DD_API_KEY` provided.